mParticle-Apple-SDK/Include/mParticle.h

```diff
@@ -980,6 +982,7 @@ Defaults to false. Prevents the eventsHost above from overwriting the alias endp
  */
```

Comment:
Should we move the ifndef to above this comment (right under the `#if TARGET_OS_IOS = 1` line) to exclude this option entirely? Or would that cause other problems? I assume right now it just won't do anything if location is disabled?